src/playlist.js

```diff
@@ -91,7 +107,8 @@
     // available PTS information
     for (left = range.start; left < range.end; left++) {
       segment = playlist.segments[left];
-      if (segment.minVideoPts !== undefined) {
+      if (segment.minVideoPts !== undefined ||
```

Comment:
Still getting error `TypeError: Cannot read property 'minVideoPts' of undefined`